crates/biome_service/src/settings.rs

```diff
@@ -858,13 +858,7 @@ impl VcsIgnoredPatterns {
         };
 
         let nested_ignored = nested.iter().any(|gitignore| {
-            let ignore_directory = if gitignore.path().is_file() {
-                // SAFETY: if it's a file, it always has a parent
-                gitignore.path().parent().unwrap()
-            } else {
-                gitignore.path()
-            };
```

Comment:
Based on doc comments, `gitignore.path()` is always a directory path where contains the `.gitignore` file.

Reply:
I hope you're right :sweat_smile: This may be something @ematipico can say better than I can.

Reply:
It's a good change. I initially added this because I didn't understand completely how the library works. But now we always store folders, so this piece of code can go away ☺️